backend/config/corpora-api.yml

```diff
@@ -1475,9 +1475,11 @@ components:
           A single-element object with the ontology term id as the element's property name and the ontology term label
           as the element's property value.
         type: object
-        maxProperties: 1
-        additionalProperties:
-          type: string
+        # TODO: fix
```

Comment:
future pr?

Reply:
All TODOs I introduced are now linked to future stories.